contracts/contracts/coordination/TACoChildApplication.sol

```diff
@@ -178,6 +229,14 @@ contract TACoChildApplication is ITACoRootToChild, ITACoChildApplication, Initia
             mstore(activeStakingProviders, resultIndex)
         }
     }
+
+    // TODO only for backward compatibility
+    function getActiveStakingProviders(
```

Comment:
While this maintains backwards compatibility for now - presumably we will need to updates to `nucypher/nucypher` and `nucypher/nucypher-porter` for proper sampling. 

Particularly in Porter's case, the `/bucket_sampling` and `/get_ursulas` (although not sure we still need this one) endpoint(s) will need to take a duration.